Refreshing reference (#2039)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/index.md

title: Reference overview
---

> # Reference 

Reviewer: To match the first topics in both Guides and Concepts, this should be titled:
# Reference overview

Details on parameters, requests, and responses for Okta's API endpoints.

> ## Core Okta APIs

Reviewer 2: ```suggestion
## Core Okta API
```

> Core Okta API is the primary way that apps and services interact with Okta. You can use it to sign in your users and manage Okta objects.

Reviewer 2: ```suggestion
The Core Okta API is the primary way that apps and services interact with Okta. You can use it to implement basic auth functions such as signing in your users and programmatically managing your Okta objects. 
```

[Explore core Okta API](/docs/reference/core-okta-api/)

> <Cards><Card href="/docs/concepts/authentication/" :showHeaderIcon=false cardTitle="Postman Collections" :showFooter=true>Import any Okta API collection for Postman.</Card><Card href="/docs/concepts/oauth-openid/" :showHeaderIcon=false cardTitle="Advanced Service Access API" :showFooter=true>Scale your controls of servers via automation.</Card></Cards>

Reviewer 2: ```suggestion
<Cards><Card href="/docs/concepts/authentication/" :showHeaderIcon=false cardTitle="Postman Collections" :showFooter=true>Import any Okta API collection for Postman.</Card><Card href="/docs/concepts/oauth-openid/" :showHeaderIcon=false cardTitle="Advanced Service Access API" :showFooter=true>Scale your control of servers with automation.</Card></Cards>
```

> <HorizontalDivider/>

Reviewer: We don't use horizontal dividers anywhere else in the site. Is there a reason that these were added here and in line 26?

Author: It was present on designs here in Figma, so added them. 

https://www.figma.com/file/2ebFMRMittnBfnuOtNEJ3T/API-Reference-and-Release-Notes-(Dev-Doc)-IA-Clean-Up?node-id=1017%3A0

Reviewer: Whose figma design is that - so can we discuss with the design/UX person?

## Inline Hooks 

> Inline hooks allow you to integrate custom functionality into specifc Okta process flows. 

Reviewer 2: ```suggestion
Inline Hooks allow you to integrate custom functionality into specific Okta process flows. 
```

> For example, you can migrate users from another data store and keep user’s current password with [Password Inline Hook](/docs/reference/password-hook/).

Reviewer 2: ```suggestion
For example, you can migrate users from another data store and keep the user’s current password with a [Password Inline Hook](/docs/reference/password-hook/).
```

> <HorizontalDivider/>

Reviewer: See line 18

## Okta Expression Language
Expressions allow you to reference, transform, and combine attributes before you store or parse them.

> [Learn more](/docs/reference/okta-expression-language/)

Reviewer: We seem to have skipped mention of the SCIM reference info. Please add the following text:

## System for Cross-domain Identity Management
SCIM is an industry-standard protocol for automating the exchange of user identity information and is part of the Okta Lifecycle Management feature. For example, as your company onboards employees, new user accounts are created in your application so they can connect immediately. Okta supports SCIM versions 1.1 and 2.0.

[Learn more](/docs/reference/scim/)

<Cards><Card href="/docs/concepts/authentication/" :showHeaderIcon=false cardTitle="Postman Collections" :showFooter=true>Import any Okta API collection for Postman.</Card><Card href="/docs/concepts/oauth-openid/" :showHeaderIcon=false cardTitle="Advanced Service Access API" :showFooter=true>Scale your control of servers with automation.</Card></Cards>

## Inline Hooks

Inline Hooks allow you to integrate custom functionality into specific Okta process flows.
For example, you can migrate users from another data store and keep the user’s current password with a [Password Inline Hook](/docs/reference/password-hook/).

## Okta Expression Language

Expressions allow you to reference, transform, and combine attributes before you store or parse them.

[Learn more](/docs/reference/okta-expression-language/)

> ## Okta Expression Language

Reviewer: This title should be:
"System for Cross-domain Identity Management"
